src/appstudio_context.ts

```diff
@@ -39,18 +41,22 @@ function extractCfSubAccountAndWs(wsBaseUrlString: string, context: AppStudioMul
 export function createContextObject(): AppStudioMultiContext {
   // get the user name from the env
   const userName = getEnv(USER_NAME, "Feature toggle env USER_NAME was NOT found in the environment variables");
+  // get the environment from the env
+  const environment = getEnv(LANDSCAPE_ENVIRONMENT, "Feature toggle env LANDSCAPE_ENVIRONMENT was NOT found in the environment variables");
```

Comment:
They exposed environment variables in small letters and you have it in big. Doesnt it fail the flow?

Reply:
I have talked to them and it was changed to upper case

Reply:
yep, I saw :)